tapiriik/local_settings.py.example

```diff
@@ -7,6 +7,9 @@ DROPBOX_FULL_APP_SECRET = "####"
 DROPBOX_APP_KEY = "####"
 DROPBOX_APP_SECRET = "####"
 
+GOOGLEDRIVE_CLIENT_ID = "####"
```

Comment:
Is this a special google drive credential? Or just a google app credential (thinking about when I get around to doing Google Fit).